### ISSUE TITLE
Inspector crashes when Debug + LiveSync command executes on iOS device

### ISSUE THREAD

**Issue by vchimev** [NativeScript/ios-runtime#626]
# How to reproduce

A connected device is required, for example:
```
mcsofvchimev:appJS vchimev$ tns device
┌───┬─────────────────────────┬──────────┬──────────────────────────────────────────┬────────┬───────────┐
│ # │ Device Name             │ Platform │ Device Identifier                        │ Type   │ Status    │
│ 1 │ iPhone 5s iOS 93 Device │ iOS      │ ccd596fa85d0c0c78c9b5a95147c1ba49c9dcd67 │ Device │ Connected │
└───┴─────────────────────────┴──────────┴──────────────────────────────────────────┴────────┴───────────┘
```

Command to reproduce:
```
tns debug ios --watch
```

# Result

NativeScript Inspector fails to load the resources.

Output: 
```
mcsofvchimev:appJS vchimev$ tns debug ios --watch
Project successfully prepared (ios)
Transferring project files...
Successfully transferred all files.
Applying changes...
Mounting /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/DeviceSupport/9.3 (13E230)/DeveloperDiskImage.dmg
[Mounting] LookingUpImage
Successfully run application org.nativescript.appJS on device with ID ccd596fa85d0c0c78c9b5a95147c1ba49c9dcd67.
2016-08-16 11:43:52.781 appJS[1251:644979] Unzipping LiveSync folder. This could take a while...
2016-08-16 11:43:52.802 appJS[1251:644979] Unzipped 8 entries in 15.855014ms.
2016-08-16 11:43:52.803 appJS[1251:644979] tns_modules folder not livesynced. Using tns_modules from the already deployed bundle...
NativeScript loaded bundle file:///System/Library/Frameworks/UIKit.framework
NativeScript loaded bundle file:///System/Library/Frameworks/Foundation.framework

Setting up proxy...
Press Ctrl + C to terminate, or disconnect.

Successfully synced application org.nativescript.appJS on device ccd596fa85d0c0c78c9b5a95147c1ba49c9dcd67.
NativeScript loaded bundle file:///System/Library/Frameworks/CoreImage.framework
NativeScript loaded bundle file:///System/Library/Frameworks/QuartzCore.framework
NativeScript loaded bundle file:///System/Library/Frameworks/iAd.framework
NativeScript loaded bundle file:///System/Library/Frameworks/AVFoundation.framework
NativeScript loaded bundle file:///System/Library/Frameworks/MapKit.framework
NativeScript loaded bundle file:///System/Library/Frameworks/SceneKit.framework
NativeScript loaded bundle file:///System/Library/Frameworks/QuickLook.framework
NativeScript loaded bundle file:///System/Library/Frameworks/CoreSpotlight.framework
Frontend client connected.
Backend socket created.
2016-08-16 11:43:53.601 appJS[1251:644994] NativeScript debugger attached.
NativeScript loaded bundle file:///var/containers/Bundle/Application/21707444-174E-470E-8C18-F225082749B3/appJS.app/Frameworks/TNSWidgets.framework
NativeScript loaded bundle file:///System/Library/Frameworks/CoreGraphics.framework
***** Fatal JavaScript exception - application has been terminated. *****
Native stack trace:
1   0x10040c87c -[TNSRuntimeInspector reportFatalError:]
2   0x10008a7cc TNSInspectorUncaughtExceptionHandler
3   0x180b4f138 <redacted>
4   0x1801b423c <redacted>
5   0x1801a6f44 <redacted>
6   0x1801a6b10 __cxa_rethrow
7   0x180173110 std::__1::__call_once(unsigned long volatile&, void*, void (*)(void*))
8   0x100416c44 Inspector::cachedResources(NativeScript::GlobalObject&)
9   0x10041ede0 Inspector::InspectorPageAgent::getResourceTree(WTF::String&, WTF::RefPtr<Inspector::Protocol::Page::FrameResourceTree>&)
10  0x100454f84 Inspector::PageBackendDispatcher::getResourceTree(long, WTF::RefPtr<Inspector::InspectorObject>&&)
11  0x100453c40 Inspector::PageBackendDispatcher::dispatch(long, WTF::String const&, WTF::Ref<Inspector::InspectorObject>&&)
12  0x100628abc Inspector::BackendDispatcher::dispatch(WTF::String const&)
13  0x10040c714 -[TNSRuntimeInspector dispatchMessage:]
14  0x10008aa14 __TNSEnableRemoteInspector_block_invoke_6
15  0x180b04f84 <redacted>
16  0x180b048bc <redacted>
17  0x180b02820 <redacted>
18  0x180a2cc50 CFRunLoopRunSpecific
19  0x182314088 GSEventRunModal
20  0x185d16088 UIApplicationMain
21  0x1008e0044 ffi_call_SYSV
22  0x1008dea28 ffi_call_int
23  0x1008de540 ffi_call
24  0x1003cc67c NativeScript::FFICall::call(JSC::ExecState*)
25  0x1006bbde8 JSC::LLInt::setUpCall(JSC::ExecState*, JSC::Instruction*, JSC::CodeSpecializationKind, JSC::JSValue, JSC::LLIntCallLinkInfo*)
26  0x1006c2368 llint_entry
27  0x1006c2378 llint_entry
28  0x1006bc6c8 vmEntryToJavaScript
29  0x100668eb0 JSC::JITCode::execute(JSC::VM*, JSC::ProtoCallFrame*)
30  0x10064e84c JSC::Interpreter::executeCall(JSC::ExecState*, JSC::JSObject*, JSC::CallType, JSC::CallData const&, JSC::JSValue, JSC::ArgList const&)
31  0x10077993c JSC::call(JSC::ExecState*, JSC::JSValue, JSC::CallType, JSC::CallData const&, JSC::JSValue, JSC::ArgList const&, WTF::NakedPtr<JSC::Exception>&)
JavaScript stack trace:
1   UIApplicationMain@file:///app/tns_modules/application/application.js:233:26
2   start@file:///app/tns_modules/application/application.js:233:26
3   anonymous@file:///app/app.js:2:18
4   evaluate@[native code]
5   moduleEvaluation@[native code]
6   @[native code]
7   promiseReactionJob@[native code]
JavaScript error:
file:///app/tns_modules/application/application.js:233:26: JS ERROR Error: *** -[__NSCFString substringFromIndex:]: Index 117 out of bounds; string length 97
```

**Comment by jasssonpet** [comment]
This seems to be related with the symlinked `tns_modules` folder during livesync.

**Comment by KristinaKoeva** [comment]
Fixed with #634
